python-requirements-apple-silicon.txt

```diff
@@ -0,0 +1,123 @@
+airium==0.2.6
```

Comment:
## Alternate file for Apple Silicon
This issue explains the problem and the solution implemented:
- #381